### PR TITLE
DRF Integration basics + API for registration & volunteer list

### PR DESCRIPTION
**Pull request by stavlocker**
- Added new app, api, which serves the Django REST Framework (will eventually be the only app in use)
 - Added four available APIs for registration, volunteer list (only for authenticated users), register, send verification code & verify verification code (last two are currently stubs, TBD in #198 )
 - Fixed wanted assignment choices

### CODE REVIEW

File: api/serializers.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+from client.models import Volunteer, VolunteerSchedule, City
+from rest_framework import serializers
+
+
+class RegistrationSerializer(serializers.HyperlinkedModelSerializer):
+    city = serializers.PrimaryKeyRelatedField(queryset=City.objects.all())
+
+    class Meta:
```
Comment (astar26) [inline api/serializers.py:8]: Why is this not using the modern class syntax?

Comment (astar26) [reply api/serializers.py:8]: Also, add optional fields for minors and for terms agreement.

Comment (stavlocker) [reply api/serializers.py:8]: > 
> 
> Why is this not using the modern class syntax?

That's how it's like all over the DRF documentation.



> 
> 
> Also, add optional fields for minors and for terms agreement.

Terms agreement would be frontend, I'm adding the minors fields.

Comment (astar26) [reply api/serializers.py:8]: Possibly a separate issue, but we'd like to mark those who did not approve (yet), with a mechanism for backwards approval. Will be added as a separate issue.

**Pull request merged**